src/main/scala/io/iohk/ethereum/validators/Validators.scala

```diff
@@ -0,0 +1,19 @@
+package io.iohk.ethereum.validators
+
+trait Validators {
+
+  val blockValidator: BlockValidator
+  val blockHeaderValidator: BlockHeaderValidator
+  val ommersValidator: OmmersValidator
+  val signedTransactionValidator: SignedTransactionValidator
+
+}
+
+object ValidatorsImpl extends Validators {
+
+  override val blockValidator: BlockValidator = BlockValidator
+  override val blockHeaderValidator: BlockHeaderValidator = BlockHeaderValidator
+  override val ommersValidator: OmmersValidator = OmmersValidator
+  override val signedTransactionValidator: SignedTransactionValidator = SignedTransactionValidator
+
+}
```

Comment:
I would think this can be moved to 'NodeBuilder'. 
```ValidatorsBuilder {
 val validators = new Validators { val blockValidator = BlockValidator ... } ```
Don't hold up the merge, but if you've got to do one more commit, please consider moving this...